main.py

```diff
@@ -127,9 +149,12 @@ def render():
             if event.key == pygame.K_s:
                 if not os.path.isdir("images"):
                     os.makedirs("images")
+                lines_to_svg(lines, f"images/{datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}.svg");
                 pygame.image.save(screen, f"images/{datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}.png")
             if event.key == pygame.K_a:
                 show_axes = not show_axes
+            if event.key == pygame.K_p:
+                paused = not paused

```

Comment:
When paused, the animation pauses but if I click somewhere while it is paused, it spawns a planet there which isn't visible while it is still paused, but after resuming it, you can see the spawned planet entering the animation.